core/src/i18n/en.yaml

```diff
@@ -31,11 +31,11 @@ api-rules:
       required-scope: Required Scope
       types: Types
     messages:
-      one-method: This access strategy requires at least one method.
-      one-strategy-required: At least one strategy is required.
+      one-method: This access strategy requires at least one method
```

Comment:
I guess I'd use punctuation here, since it's a full sentence, but I'd be happy to see this message in UI. Let's discuss.